src/scc_hypervisor_collector/cli/__init__.py

```diff
@@ -5,11 +5,55 @@
 import logging
 import os
 import sys
-from typing import (Optional, Sequence)
+from logging.handlers import RotatingFileHandler
+from typing import (Any, Optional, Sequence)
 import yaml
 
 from scc_hypervisor_collector import __version__ as cli_version
 from scc_hypervisor_collector import (ConfigManager, CollectionScheduler)
+from scc_hypervisor_collector.api.exceptions import (
```

Comment:
So all of the exceptions should be available as exports from the base scc_hypervisor_collector.api itself, so you should import from there rather than from the `.exceptions`. makes the CLI implementation agnostic of the internal layout/structure of the api sub-package.

```suggestion
from scc_hypervisor_collector.api import (
```